examples/chart/teleport/README.md

```diff
@@ -0,0 +1,107 @@
+# Running Teleport locally on minikube
+
+Run:
+
+```
+./scripts/install-on-minikube
+```
+
+Type your desired password, capture the barcode with your MFA device like Google Authenticator, type the OTP.
+
+# Running Teleport on Kubernetes
+
+- Download the `license.pem` from the Teleport dashboard
+
+```
+# Rename the license.pem to the filename that this chart expects
+cp ~/Downloads/license.pem license-enterprise.pem
+
+kubectl create secret generic license --from-file=license-enterprise.pem
+
+# No need to include tls assets here, because we terminate TLS at ngrok this time
```

Comment:
Remove reference for ngrok here and below

Reply:
👍